bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/api/CatalogQuery.scala

```diff
@@ -18,6 +18,21 @@ case class MagnitudeQueryFilter(mc: MagnitudeConstraints) extends QueryResultsFi
   def filter(t: SiderealTarget): Boolean = mc.filter(t)
 }
 
+sealed abstract class CatalogName(val id: String, val displayName: String)
+
+case object sdss extends CatalogName("sdss9", "SDSS9 @ Gemini")
+case object gsc234 extends CatalogName("gsc234", "GSC234 @ Gemini")
+case object ppmxl extends CatalogName("ppmxl", "PPMXL @ Gemini")
+case object ucac4 extends CatalogName("ucac4", "UCAC4 @ Gemini")
+case object twomass_psc extends CatalogName("twomass_psc", "TwoMass PSC @ Gemini")
+case object twomass_xsc extends CatalogName("twomass_xsc", "TwoMass XSC @ Gemini")
+case object simbad extends CatalogName("simbad", "Simbad")
+
```

Comment:
Good reminder, I have the feeling I was already hit by that issue and I switched them to uppercase but clearly I reverted